components/brave_federated/task/model_unittest.cc

```diff
@@ -0,0 +1,112 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_federated/task/model.h"
+#include <cstddef>
+
+#include "brave/components/brave_federated/util/linear_algebra_util.h"
+#include "testing/gtest/include/gtest/gtest.h"
+#include "third_party/googletest/src/googletest/include/gtest/gtest.h"
+
+// npm run test -- brave_unit_tests --filter=BraveFederatedLearning*
+
+namespace brave_federated {
+
+ModelSpec g_k_model_spec = {
```

Comment:
See Chromium style guide for naming/use of globals

Reply:
mhm is it just kModelSpec? As per https://google.github.io/styleguide/cppguide.html#Constant_Names. This naming formatting was suggested by the clangd